app/uk/gov/hmrc/traderservices/models/TraderServicesFileTransferRequest.scala

```diff
@@ -30,7 +30,8 @@ case class TraderServicesFileTransferRequest(
   fileMimeType: String,
   batchSize: Int,
   batchCount: Int,
-  correlationId: Option[String] = None
+  correlationId: Option[String] = None,
+  requestId: Option[String] = None
```

Comment:
OBSERVATION: Might be worth a comment explaining that these values are not expected to be supplied in the incoming JSON (that is deserialized into a `TraderServicesFileTransferRequest` instance)